### PR TITLE
BeSman updated with new 'bes modify' command

### PR DESCRIPTION
**Pull request by jobyko**
@asa1997 
Please share your comments.

### CODE REVIEW

File: src/main/bash/besman-modify.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+function __bes_mod
```
Comment (asa1997) [inline src/main/bash/besman-modify.sh:3]: Since command is modify this function name should be __bes_modify

File: src/main/bash/besman-modify.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ {
+
+ local playbookdir="$HOME/$BESMAN_PLAYBOOK_REPO"
+ local filename=$1
+
+ if [[ ( -d "$playbookdir" ) && ( -f $playbookdir/$filename ) ]]; then
+         cd $playbookdir
+
+         __besman_open_file $playbookdir || return 1
+         __besman_git_stage $filename || return 1
```
Comment (asa1997) [inline src/main/bash/besman-modify.sh:13]: I dont think these steps are necessary here. All these are handled by save command.

Comment (asa1997) [reply src/main/bash/besman-modify.sh:13]: Modify should only help the user to modify the playbook he wishes.

**Review by asa1997 — CHANGES_REQUESTED** [review]
@jobyko Please address the comments.

File: src/main/bash/besman-modify.sh (hunk 2 above)
Comment (jobyko) [reply src/main/bash/besman-modify.sh:13]: @asa1997 
Should we consider keeping, __besman_open_file to open file for edit.

**Review by asa1997 — APPROVED** [review]
@jobyko @panickervinod LGTM.

**Comment by panickervinod** [comment]
@jobyko @asa1997 can this be called update rather than modify.?

File: src/main/bash/besman-modify.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+function __bes_modify
```
Comment (panickervinod) [inline src/main/bash/besman-modify.sh:3]: @jobyko can this be renamed as __bes_modify to __bes_update ? do you see a name clash?

**Pull request closed without merging**